smtpapi.go

```diff
@@ -189,13 +189,16 @@ func escapeUnicode(input string) string {
 			// surrogate pair
 			var r1, r2 = utf16.EncodeRune(r)
 			var s = fmt.Sprintf("\\u%x\\u%x", r1, r2)
-			buffer.WriteString(s)
+			_, err := buffer.WriteString(s)
+			_ = err // error always nil https://golang.org/pkg/bytes/#Buffer.WriteString
 		} else if r > 127 {
 			var s = fmt.Sprintf("\\u%04x", r)
-			buffer.WriteString(s)
+			_, err := buffer.WriteString(s)
+			_ = err // error always nil https://golang.org/pkg/bytes/#Buffer.WriteString
 		} else {
 			var s = fmt.Sprintf("%c", r)
-			buffer.WriteString(s)
+			_, err := buffer.WriteString(s)
+			_ = err // error always nil https://golang.org/pkg/bytes/#Buffer.WriteString
```

Comment:
would this be better as a `//nolint: ...` directive? It's not clean, but neither two-step discard.

Reply:
That's a great idea @extemporalgenome . I didn't know about `//nolint:` I'll make that change shortly.